sdk/python/packages/flet-cli/src/flet_cli/commands/build.py

```diff
@@ -49,28 +47,6 @@ class Command(BaseCommand):
     def __init__(self, parser: argparse.ArgumentParser) -> None:
         super().__init__(parser)
 
-        self.pubspec_path = None
-        self.rel_out_dir = None
-        self.assets_path = None
-        self.package_platform = None
-        self.target_platform = None
-        self.flutter_dependencies = None
-        self.package_app_path = None
-        self.options = None
-        self.pubspec = None
-        self.template_data = None
-        self.python_module_filename = None
-        self.out_dir = None
-        self.python_module_name = None
-        self.get_pyproject = None
-        self.python_app_path = None
-        self.no_rich_output = None
-        self.emojis = {}
-        self.dart_exe = None
-        self.verbose = False
-        self.build_dir = None
-        self.flutter_dir: Optional[Path] = None
-        self.flutter_exe = None
```

Comment:
why remove this?

Reply:
It makes type checker really "red":

<img width="602" alt="image" src="https://github.com/user-attachments/assets/137a3891-b5c5-4319-bd27-2d06977ebf27">

Reply:
The red lines you are showing seem to come from using `self.options.X` instead? 

Without that block of initializations i get: 
![image](https://github.com/user-attachments/assets/7cbe3ec2-06a3-411f-81e2-7f249c0eb0ed)

SO discussion: https://stackoverflow.com/a/19292653 (common python practice)

Reply:
Looks like it's coming from `self.options`. What would be the solution?

<img width="859" alt="image" src="https://github.com/user-attachments/assets/4eb5ab36-7b1b-4d14-bfdd-9ec085953b2f">

Reply:
Perhaps use `cast` as you did in the others? 
Or perhaps just mute the warning?